.gitmodules

```diff
@@ -1,27 +0,0 @@
-# We shall favor using specific git tags or commit hashes to ensure code stability;
-# Try to avoid depending on branches whose code may change unexpectedly.
-
-[submodule "packages/protocol/lib/openzeppelin-contracts-upgradeable"]
-	path = packages/protocol/lib/openzeppelin-contracts-upgradeable
-	url = https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable
-	tag = v4.8.2
-
-[submodule "packages/protocol/lib/openzeppelin-contracts"]
-	path = packages/protocol/lib/openzeppelin-contracts
-	url = https://github.com/OpenZeppelin/openzeppelin-contracts
-	tag = v4.8.2
-
-[submodule "packages/protocol/lib/forge-std"]
```

Comment:
you should also remove `.git/modules/packages` but not sure this is actually part of the git history.

Reply:
i updated the PR description. we will need to execute a command:
```
git submodule deinit -f --all 
git clean -fd
```
after pulling the changes from main.